src/licensedcode/models.py

```diff
@@ -737,6 +747,17 @@ def validate(licenses, verbose=False, no_dupe_urls=False):
         return errors, warnings, infos
 
 
+def get_yaml_safe_text(text):
+
+    data = {"text": text}
```

Comment:
We are doing more than this and hence it is expensive. The YAML load was not failing just because of the empty newlines but: both empty newlines and different indentation levels (?) being present. So we only pad newlines with space on failing load(). Also considering other options here.